doc/ale.txt

```diff
@@ -573,6 +573,29 @@ g:ale_change_sign_column_color                 *g:ale_change_sign_column_color*
   windows.
 
 
+g:ale_cmd_wrapper                                           g:ale_cmd_wrapper
```

Comment:
You want to put `*g:ale_cmd_wrapper*` on the right margin, and make sure it lines up with everything else. That creates a help tag for looking up `:help g:ale_cmd_wrapper`.